content/momentum/4/using-dkim.md

```diff
@@ -1,13 +1,13 @@
 ---
-lastUpdated: "03/26/2020"
+lastUpdated: "08/02/2023"
 title: "Using DomainKeys Identified Mail (DKIM) Signatures"
-description: "Domain Keys Identified Mail DKIM is a mechanism that allows verification of the source and contents of email messages Using DKIM sending domains can include a cryptographic signature in outgoing email messages A message's signature may be verified by any or all MT As during transit and by the Mail..."
+description: "Domain Keys Identified Mail DKIM lets a receiving email server verify of the source and content of an email message. This is an anti-spoofing (and anti-phishing) measure. Using DKIM sending domains can include a cryptographic signature in outgoing email messages A message's signature may be verified by any or all MT As during transit and by the Mail..."
```

Comment:
_lets a receiving email server verify of the source and content of an email message._

Maybe it's necessary to remove the `of`? `verify the source and content`